packages/sdk-v2/src/hooks/useBuyGd.ts

```diff
@@ -0,0 +1,44 @@
+import { Contract } from "ethers";
```

Comment:
this file and the swaphelper should be in an sdk folder, same structure like other sdks